.travis.yml

```diff
@@ -18,4 +18,4 @@ script:
 after_success:
   - cat ./packages/mobx-state-tree/coverage/lcov.info|./packages/mobx-state-tree/node_modules/coveralls/bin/coveralls.js
 node_js:
-  - 9.3.0
+  - "node"
```

Comment:
what does this mean?

Reply:
latest lts version of node

Reply:
I mean make Travis use the latest lts version of node, a VM with node 10.x and npm 6 installed in this case